### ISSUE TITLE
Is Heroku dead for OSS? Remove the Heroku example and CI

### ISSUE THREAD

**Issue by aantron** [aantron/dream#306]
As I understand it, it's no longer possible to deploy to Heroku for free, which complicates maintenance with the need to be on a paid plan. I also suspect that Heroku is much less popular as a result of this. So, I'm inclined to remove the [Heroku example](https://github.com/aantron/dream/tree/680d5d289dbca77ec5570378aaacaa06f0306545/example/z-heroku), as it probably won't be updated. It can still be found in the repo history if anyone needs it.

**Comment by Ashvith10** [comment]
@aantron how about trying out Railway or Replit? Both of them provide a Nix-based approach, meaning that you have flexibility over choosing packages. Personally, I've struggled with using opam as I'm a beginner myself (you're forced to use only Nix and not the in-built package manager, I think?), but I've tried Replit with SvelteKit, and I think it's quite nice.

**Comment by dangdennis** [comment]
I made a template on railway for dream fwiw. https://railway.app/template/mxESzS

while railway created nixpacks, for unsupported languages, docker is still the way to go.

**Comment by aantron** [comment]
> I made a template on railway for dream fwiw. https://railway.app/template/mxESzS
> 
> while railway created nixpacks, for unsupported languages, docker is still the way to go.

Great! What's the best way for us to present it in the Dream repo? As a link? As an example within the Dream repo?

**Comment by aantron** [comment]
I think I can write a minimalistic example based on this.

**Comment by aantron** [comment]
Railway does seem to lack a free plan. The issue here is not money (Dream already pays much more to host dream.as). I don't think I can justify the time cost of managing an extra entanglement for a hosting platform at this point.

**Comment by Ashvith10** [comment]
@aantron fair point. I think I remember why I deleted my account immediately. At least from a beginner's PoV, they would prefer a free hosting service. Perhaps Replit would be a fair place to write a few tutorials? What would be a example tutorial you'd recommend me to try out?

**Comment by aantron** [comment]
We'd probably wantt just the simplest "Hello, world!" type of example, like the [Heroku one](https://github.com/aantron/dream/blob/087b5ff1eac79f103fb4ab389bc769e983f81301/example/z-heroku/app.ml):

```ocaml
let () =
  Dream.run ~interface:"0.0.0.0" ~port:80
  @@ Dream.logger
  @@ Dream.router [
    Dream.get "/" (fun _ -> Dream.html "Dream running in Reeplit!");
  ]
```
